main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	clientpkg "awscleaner/pkg/aws"
+	"fmt"
+	_ "fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/sts"
+)
+
+const (
+
+	// the credentials for aws-account-shredder
+	accessID  = ""
+	secretKey = ""
+	accountid = "" // it is 12 digit account id , the sub level account , for future use and debugging purpose
+)
+
+func main() {
+
+	// creating a new cient with us-east-1 region by default
+	client, err := clientpkg.NewClient(accessID, secretKey, "", "us-east-1")
+	if err != nil {
+		fmt.Println("error is %v", err)
+	}
+	assumedRole, err := client.AssumeRole(&sts.AssumeRoleInput{RoleArn: aws.String("arn:aws:iam::173028519319:role/OrganizationAccountAccessRole"), RoleSessionName: aws.String("agautam")})
```

Comment:
Can you remove `173028519319` from the code for now? Make it an empty const like accessID, etc.

Reply:
Session name as well

Reply:
resolved.